tests/test_render.py

```diff
@@ -104,7 +104,7 @@ def request_queue_assertions(
 
         if expected_length is not None:
             self.assertEqual(len(request_queue), expected_length)
-    
```

Comment:
Oh noes, trailing whitespace courtesy of @chriddyp ??? 😅

Reply:
no way!! 🙊 🙊 🙊